__tests__/tripRequestTest/tripRequestView.test.js

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { TripRequest, mapStateToProps } from '../../src/views/triprequest.view';
+import sinon from 'sinon';
+import { applyMiddleware, createStore } from 'redux';
+import thunk from 'redux-thunk';
+import { props } from '../../__mockData__/tripRequest-mock-data';
+import reducer from '../../src/reducers/index';
+
+import { Provider } from 'react-redux';
+
+const middlewares = [thunk];
+const testStore = state => {
+	const createStoreWithMiddleware = applyMiddleware(...middlewares)(
+		createStore,
+	);
+	return createStoreWithMiddleware(reducer, state);
+};
+const initialSetUp = (initialState = {}) => {
+	const store = testStore(initialState);
+	const wrapper = mount(
+		<Provider store={store}>
+			<TripRequest {...props} />
+		</Provider>,
+	);
+	return wrapper;
+};
+const setUp = (initialState = {}) => {
+	const store = testStore(initialState);
+	const wrapper = shallow(<TripRequest {...props} store={store} />);
+	return wrapper;
+};
+
+describe('Trip Request component tests', () => {
+	it('should render Trip Request view page successfully', () => {
+		const wrapper = shallow(<TripRequest />);
+		expect(wrapper.find('div').length).toBe(0);
+	});
+	it('should handle change successfully', () => {
+		const component = initialSetUp();
+		component
+			.find('Connect(WithStyles(OneWayForm))')
+			.props()
+			.onHandleChange({ target: { name: 'To', value: 'hsjhjhsj' } });
+	});
+	it('should handle change successfully when destination is Provided', () => {
```

Comment:
@Jajabenit250 kindly some assertion here too. Also, update elsewhere it's applicable.